.replit

```diff
@@ -1,2 +1,10 @@
+run = "npm start"
+entrypoint = "server.js"
+
+[packager]
 language = "nodejs"
-run = "npm start"
\ No newline at end of file
+
+  [packager.features]
```

Comment:
Do we need this spacing?

Reply:
It just how it is formatted in the Replit docs.